docs/100.yaml

```diff
@@ -1,3 +1,20 @@
+# This file tracks the formalisation status of theorems from Freek Wiedijk's "Formalizing 100
+# Theorems" list at https://www.cs.ru.nl/~freek/100/
+#
+# This file, contains for each theorem in the list,
+# - an entry, labelled by its number,
+# - |title|: a human-readable title
+# - (optional) |decl|: if a theorem is formalised in mathlib, the archive or counterexamples,
+#     the name of the corresponding declaration
+# - (optional) |decls|: like |decl|, but a list of declarations
+#     (if one theorem is split into multiple declarations)
+# - (optional) |doc_decls|: ...
```

Comment:
I wasn't sure what `doc_decls` was for - this needs to be updated before we merge (here and in 1000.yaml as well).